main.go

```diff
@@ -299,13 +299,16 @@ func loadConfig() (*config, error) {
 		return nil, err
 	}
 
-	configFilePath := os.Getenv("PACKER_CONFIG")
-	if configFilePath != "" {
+	// start by loading from PACKER_CONFIG if available
+	var configFilePath string
+	if v := os.Getenv("PACKER_CONFIG"); v != "" {
```

Comment:
Why use "v" at all as an intermediary? Seems like you could just directly use configFilePath and skip the "else" altogether.

Reply:
You are right. It's not needed. Nice catch :+1: